src/App.js

```diff
@@ -1,27 +1,36 @@
-// import logo from './logo.svg';
 import './App.css';
-import movieData from './movieData'
+import apiData from './apiCalls'
 import MovieCard from './movieCard'
 import Movies from './movies'
 import React, { Component } from 'react'
 import Header from './Header'
 import FeatureDisplay from './FeatureDisplay'
 import MovieInfo from './MovieInfo'
 
+
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      movieData: movieData.movies,
+      movieData: [],
       selectedMovie: null
     }
   }
 
+  componentDidMount = () => {
+    apiData.allMovieData()
+    .then(data => this.setState({movieData: data.movies}))
+    .catch((error) => {
+      console.log('Error:', error)
```

Comment:
Do you have any plans for this error down the road - maybe storing it in state so we can eventually surface it to the user?